httpclient/httpclient.go

```diff
@@ -132,12 +132,6 @@ func (c *Client) Call(ctx context.Context, r Request) (err error) {
 	}
 	u.RawQuery = r.Query.Encode() // returns "" if Query is nil
 
-	ctx, span := o11y.StartSpan(ctx, spanName)
-	defer o11y.End(span, &err)
-	span.AddRawField("span.kind", "httpclient")
```

Comment:
i think this is used in a few queries?

Reply:
found a couple: https://ui.honeycomb.io/circleci/board/zhPAiyhHmMW/Output, https://ui.honeycomb.io/circleci/datasets/execution/result/F4pNCguNyp1/a/jTZNUPXR1bU/Outgoing-HTTP-Requests